Testing/functional/tests/cypress/integration/ui/vaccineCard/publicCovidTest.js

```diff
@@ -0,0 +1,419 @@
+const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December",
+];
+
+const dobYearSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectYear]";
+const dobMonthSelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectMonth]";
+const dobDaySelector =
+    "[data-testid=dateOfBirthInput] [data-testid=formSelectDay]";
+const dovYearSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectYear]";
+const dovMonthSelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectMonth]";
+const dovDaySelector =
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectDay]";
```

Comment:
The public test page shouldn't have a date of vaccination.  I've created bug [AB#11901](https://dev.azure.com/qslvic/Health%20Gateway/_workitems/edit/11901) to track the issue.